frontend/src/app/organization/widgets/organization-details-info-card/organization-details-info-card.widget.html

```diff
@@ -6,15 +6,15 @@
   <mat-card-header class="organization-header">
     <div class="left-header-info" *ngIf="!isTablet && !isHandset">
```

Comment:
How thorough are we trying to be on migration of `if` syntax?

Reply:
Just migrated everything remaining in the organizations feature! This widget needs to be reworked anyway (the `isTablet` and `isHandset` usage seems suspect), but that can be done in the material refactor.